nodes/persistence-node/src/config/IpfsConfig.ts

```diff
@@ -1,8 +1,19 @@
+import { isValidUrl } from "../isValidUrl";
+
 export class IpfsConfig {
   ipfsProvider = process.env.IPFS_PROVIDER ?? "http://localhost:5001";
```

Comment:
I don't think this is used. If not, you can use this instead of "ipfsUrl", defaults to undefined
Also, naming it externalIpfsProvider is best, I think, to indicate there's also an internal option

Reply:
Fixed.